packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/index.tsx

```diff
@@ -141,7 +148,10 @@ class TransactionsContainer extends React.PureComponent<Props> {
     } = this.props
     const { coin } = computedMatch.params
     const { coinfig } = window.coins[coin]
-    const interestEligibleCoin = interestEligible[coin] && interestEligible[coin]?.eligible
+    const interestEligibleCoin =
+      interestEligible && interestEligible[coin] && interestEligible[coin]?.eligible
+    const isEarnButtonEnabled = isGoldTier && interestEligibleCoin
+    const buttonResponsiveWidth = isMobile() ? 'auto' : '100px'
```

Comment:
please use a styled component for CSS rules

Reply:
This is fixed. Thanks